tests/gatys_et_al_2017/test_core.py

```diff
@@ -39,3 +39,47 @@ def test_gatys_et_al_2017_nst_smoke(subtests, mocker, content_image, style_image
 
     with subtests.test("postprocessor"):
         assert isinstance(postprocessor, type(utils.gatys_et_al_2017_postprocessor()))
+
+
+def test_gatys_et_al_2017_guided_nst_smoke(
+    subtests, mocker, content_image, style_image, content_guides, style_guides
+):
+    mock = mocker.patch(
+        "pystiche_papers.gatys_et_al_2017.core.default_image_pyramid_optim_loop"
+    )
+    styles = {}
+    for key in style_guides.keys():
+        styles[key] = (style_image, style_guides[key])
```

Comment:
1. I would call it `style_images_and_guides` to match the function signature
2.

```suggestion
    style_images_and_guides = {
        region: (style_image, guide) for region, guide in style_guides.items()
    }
```